packages/@n8n/task-runner-python/justfile

```diff
@@ -0,0 +1,20 @@
+run:
+    uv run python src/main.py
+
+sync:
+    uv sync
+
+lint:
+    uv run ruff check
+
+lintfix:
+    uv run ruff check --fix
+
+format:
+    uv run ruff format
+
+test:
+    @echo "No tests yet"
+
+typecheck:
+    uv run ty check src/
```

Comment:
`ty` is pre-release but I'd like to give it a chance to stay in the Astral toolchain. If still too green, I'll fall back to `mypy`.

Reply:
Sounds like a good idea to at least go first with it. If we encounter major blockers or issues we can fallback to others